### PR TITLE
Invalidate DL_NUM containing all zeros

### PR DESCRIPTION
**Pull request by AmoghN**
Changed `isValid` to return as `false` if DL_NUM only has zeros

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 const regs = require('./regex.json');
+const { stripZeros } = require('./utils');
```
Comment (zalsader) [inline index.js:2]: Why are we using require here, not import?

Comment (zalsader) [reply index.js:2]: Oh, this is a different library...

File: utils.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns a string where all the zeros are removed
+ */
+function stripZeros(number) {
```
Comment (zalsader) [inline utils.js:4]: Could you manually format this file? (⌥⇧F)

**Review by zalsader — APPROVED** [review]
LGTM. The library version number in jerry2 `package.json` needs to be updated after this.

File: utils.js (hunk 2 above)
Comment (AmoghN) [reply utils.js:4]: It usually does it automatically 😓

**Review by Magminder — APPROVED** [review]

**Pull request merged**